tools/config/go-templates/api_store.mustache

```diff
@@ -0,0 +1,18 @@
+ package {{packageName}}
```

Comment:
That is what is generating those stores. It is basically low effort late evening work.
If we were put more into it we can certainly make it much beter.